.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,11 +1,10 @@
+<!-- Enter details of the change here. Include additional tests that have been done, reference to the issue for tracking, etc. -->
+
+<!-- Please check the completed items below -->
 ### PR checklist
 
 - [ ] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
-- [ ] Ran the shell script under `./bin/` to update Petstore sample so that CIs can verify the change. (For instance, only need to run `./bin/{LANG}-petstore.sh`, `./bin/openapi3/{LANG}-petstore.sh` if updating the {LANG} (e.g. php, ruby, python, etc) code generator or {LANG} client's mustache templates). Windows batch files can be found in `.\bin\windows\`. If contributing template-only or documentation-only changes which will change sample output, be sure to [build the project](https://github.com/OpenAPITools/openapi-generator#14---build-projects) first.
-- [ ] Filed the PR against the [correct branch](https://github.com/OpenAPITools/openapi-generator/wiki/Git-Branches): `master`, `4.1.x`, `5.0.x`. Default: `master`.
-- [ ] Copied the [technical committee](https://github.com/openapitools/openapi-generator/#62---openapi-generator-technical-committee) to review the pull request if your PR is targeting a particular programming language.
-
-### Description of the PR
-
-(details of the change, additional tests that have been done, reference to the issue for tracking, etc)
-
+- [ ] If contributing template-only or documentation-only changes which will change sample output, [build the project](https://github.com/OpenAPITools/openapi-generator#14---build-projects) before .
+- [ ] Run the shell script(s) under `./bin/` (or Windows batch scripts under`.\bin\windows`) to update Petstore samples related to your fix. This is important, as CI jobs will verify _all_ generator outputs of your HEAD commit, and these must match the expectations made by your contribution. You only need to run `./bin/{LANG}-petstore.sh`, `./bin/openapi3/{LANG}-petstore.sh` if updating the code or mustache templates for a language (`{LANG}`) (e.g. php, ruby, python, etc).
```

Comment:
e.g. for `typescript-fetch` one should only run `bin/typescript-fetch-petstore-all.sh`, not `bin/openapi3/typescript-fetch-petstore-all.sh`

Reply:
If users aren't expected to run the openapi3 script, why does it exist? Does it hurt anything if they do run it?

Reply:
> If users aren't expected to run the openapi3 script, why does it exist? 

I don't know, probably for testing purposes?

> Does it hurt anything if they do run it?

yes, the `body` parameter changes its name and some extra inline-files are generated, leading to a failing `ensure-samples-up-to-date.sh`

Reply:
So, they're generating to the same directories? I think that's an issue if the 2.0 and 3.x outputs aren't the same.

I don't think this warrants an exception in the PR template. We should create an issue to investigate and fix for output parity. That, or generate to separate directories.